Update intro page and add missing titles

despadam opened the pull request:
Fixes #226

baffelli commented on intro.ipynb:
```diff
@@ -93,22 +81,15 @@
     "- Ask your question in the **corresponding channel**. Use `#help` for generic questions.\n",
     "- Reply **in a thread** if you want to add a comment to a question.\n",
-    "- Use `@Sasha`, `@Simone`, `@Edoardo` if you want to mention the tutorial instructors directly.\n",
+    "- Use `@Sasha`, `@Simone`, `@Despina` if you want to mention the tutorial instructors directly.\n",
```
Do we need to keep the slack info considering that we are on-site?

yakutovicha commented on intro.ipynb:
```diff
@@ -77,13 +65,13 @@
-    "- *10:50-11:10 — Break* ☕\n",
-    "- 11:10-13:00 — Topic 2\n",
-    "- 🥪 – Practice – Offline discussions on Slack"
+    "- 9:30-10:00 — Welcome coffee and breakfast\n",
```
Maybe let's just keep coffee
```suggestion
    "- 9:30-10:00 — Welcome coffee\n",
```

baffelli reviewed (commented):
Looks good to me

yakutovicha reviewed (changes requested):
Thanks, @despadam, I would only request to change the content of the cell that starts with "As you can see, the message quite clearly tells you that the solution is wrong." 

It shows the old way how results were displayed, it is not the same anymore.

despadam commented:
> Thanks, @despadam, I would only request to change the content of the cell that starts with "As you can see, the message quite clearly tells you that the solution is wrong."
> 
> It shows the old way how results were displayed, it is not the same anymore.

You are right @yakutovicha. I removed the image and updated the corresponding text.

yakutovicha reviewed (approved):
Ship it!
